src/pages/countryInfo.css

```diff
@@ -1,43 +1,43 @@
 .details-container svg {
-    margin: 0.4em;
-    font-size: xxx-large;
+  margin: 0.4em;
+  font-size: xxx-large;
 }
 
 .details-content {
-    display: flex;
+  display: flex;
   gap: 50px;
   margin: 0 2em;
   color: #fefefe;
 }
 
 .right {
-    display: flex;
-    align-items: center;
-    gap: 50px;
+  display: flex;
+  align-items: center;
+  gap: 50px;
 }
 
 .right h3 {
-    font-size:xxx-large;
+  font-size: xxx-large;
 }
 
 .details {
-    font-size: x-large;
-    line-height: 1.5;
-    font-weight: 600;
+  font-size: x-large;
+  line-height: 1.5;
+  font-weight: 600;
 }
 
 .border p {
-    display: flex;
+  display: flex;
   flex-direction: column;
   font-size: x-large;
   line-height: 1.5;
   font-weight: 600;
 }
 
 .border a {
-    color: #fefefe;
+  color: #fefefe;
 }
 
 .border a:hover {
-    background-color: #fe146d;
-}
\ No newline at end of file
+  background-color: #fe146d;
```

Comment:
- [ ] Let's always remember to follow the design the way it is. This is a mobile app with two contents (countries) side by side. We have been told to follow the design guidelines, such as typography, space between elements, colors, etc. Kindly work on the design and try as much as possible to meet the requirements.

## Original

![www behance net_gallery_31579789_Ballhead-App-(Free-PSDs)(Surface Pro 7)](https://user-images.githubusercontent.com/53124548/233781521-1070aaea-d2fb-451b-bd52-76d9474da480.png)

## YOURS

![my-world-sems onrender com_(iPhone 12 Pro)](https://user-images.githubusercontent.com/53124548/233781543-43a624a6-bdc0-4eba-9946-f4a5f15c0b49.png)